model_logging/migrations/0001_initial.py

```diff
@@ -21,7 +20,7 @@ class Migration(migrations.Migration):
                 ('date_created', models.DateTimeField(default=django.utils.timezone.now)),
                 ('operation', models.CharField(max_length=255, choices=[('added', 'Added'), ('removed', 'Removed'), ('modified', 'Modified')])),
                 ('model_path', models.CharField(max_length=255)),
-                ('data', pgcrypto.fields.TextPGPPublicKeyField(default='')),
```

Comment:
We are looking to do this without installing pgcrypto in new instances. We could follow your suggestion and provide a squash migration which bypasses that process.